controller.go

```diff
@@ -1074,7 +1078,7 @@ func (c *controller) loadDriver(networkType string) error {
 }
 
 func (c *controller) loadIPAMDriver(name string) error {
-	if _, err := plugins.Get(name, ipamapi.PluginEndpointType); err != nil {
+	if _, err := c.plugingetter.Get(name, ipamapi.PluginEndpointType, getter.LOOKUP); err != nil {
```

Comment:
c has to implement a `GetPluginGetter` to satisy `DriverAPICallback` which is a wrapper to `drvRegistry.GetPluginGetter()`. So this will be `c.GetPluginGetter()`